client/state/themes/actions/activate.js

```diff
@@ -56,6 +58,19 @@ export function activate(
 			return dispatch( showAutoLoadingHomepageWarning( themeId ) );
 		}
 
+		/**
+		 * Check if the theme is a .org Theme and not provided by .com as well (as Premium themes)
+		 * and redirect it to the Marketplace theme installation page
+		 */
+		const isDotOrgTheme = !! getTheme( getState(), 'wporg', themeId );
+		const isDotComTheme = !! getTheme( getState(), 'wpcom', themeId );
+		if ( isDotOrgTheme && ! isDotComTheme ) {
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)

This might be a beginner question or I understand it wrong. 

I assume that you added `isDotComTheme` because `isDotOrgTheme` becomes `true` when activating `.com` theme (e.g. Karuna). Why does `isDotOrgTheme` become `true`? Can `.com` theme be `.org` theme as well?

Reply:
Hi @okmttdhr, your assumption is right! 

There are themes that can be both `.org` and `.com` themes (e.g. Karuna or Twenty Twenty-Two). For those themes, we don't need [to redirect](https://github.com/Automattic/wp-calypso/pull/75546/files#diff-b5564dfc3a07fd1fe8ce1c6afc4658a6f0e6f3e88a85c7de41af2b19430f1aeeR71) to the installation page, so we need this condition to prevent that. 

In summary, we want [to redirect](https://github.com/Automattic/wp-calypso/pull/75546/files#diff-b5564dfc3a07fd1fe8ce1c6afc4658a6f0e6f3e88a85c7de41af2b19430f1aeeR71) for themes that are `.org` but not `.com`